torch/_tensor_docs.py

```diff
@@ -4657,7 +4657,7 @@ def callable(a, b) -> number
 
 add_docstr_all('index_add',
                r"""
-index_add(dim, index, tensor2) -> Tensor
+index_add(dim, index, input) -> Tensor
```

Comment:
The right name for this argument is also `index` it looks like: https://github.com/pytorch/pytorch/blob/7191dd2613051b1ddfe7f5a9cfd0397b1444a116/aten/src/ATen/native/native_functions.yaml#L5659

Reply:
Thanks, @zou3519. Fixed this. :)